app/pages/index.vue

```diff
@@ -41,7 +41,10 @@ useSeoMeta({
 <template>
   <main class="grow w-full lg:w-2/3 mx-auto mt-20 p-2">
     <SiteHeader />
-    <h3 class="text-lg leading-snug text-zinc-600 font-bold mt-8 mb-2">Open Positions</h3>
+    <h3 class="text-lg leading-snug text-zinc-600 font-bold mt-8 mb-2">
+      Open Positions
+      <span class="text-sm text-zinc-500 before:content-['('] after:content-[')']">>{{ totalPositions }}</span>
```

Comment:
```suggestion
      <span class="text-sm text-zinc-500 before:content-['('] after:content-[')']">{{ totalPositions }}</span>
```
why extra `>`?

Reply:
Also, See the following img. I want something like that and replace `Applicants` with `Positions`

![image](https://github.com/user-attachments/assets/c21cbf0e-017f-448c-9974-b2fb2729bb14)